cmd/stackdriver-prometheus-sidecar/statusz.go

```diff
@@ -11,23 +11,28 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+//go:generate statik -src=./  -include=*.html
 package main
 
 import (
 	"html/template"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"path/filepath"
 	"time"
 
+	_ "github.com/Stackdriver/stackdriver-prometheus-sidecar/cmd/stackdriver-prometheus-sidecar/statik"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/common/version"
+	"github.com/rakyll/statik/fs"
 )
 
 var (
 	serverStart = time.Now()
-	statuszTmpl = template.Must(template.ParseFiles("statusz-tmpl.html"))
+	statuszTmpl = template.New("statusz-tmpl.html")
```

Comment:
Can you move this to the `init` method? It's confusing that it gets initialized here, then updated inside `init`.

Reply:
@jkohen moved it to the init method. I have a a question using promu to build the binary. When I run "go build" in the directory "cmd/stackdriver-prometheus-sidecar", the generated binary runs as expected.  However when I use the Makefile that runs the "promu build", the binary generated is against the old version of code(I can tell because of the error message and line number). Is this WAI or I'm running promu wrong ?